packages/react-native-bridge/android/src/main/res/values/colors.xml

```diff
@@ -2,5 +2,6 @@
 <resources>
 
     <color name="status_bar_color">#006088</color>
+    <color name="white">@android:color/white</color>
```

Comment:
Without this, the build would fail as follows:

```
> Task :reBundleAarRelease
[fat-aar]target: /path/to/repo/packages/react-native-bridge/android/build/outputs/aar/@wordpress_react-native-bridge-release.aar

> Task :verifyReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':verifyReleaseResources'.
> 1 exception was raised by workers:
  com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /path/to/repo/packages/react-native-bridge/android/build/intermediates/res/merged/release/drawable/ic_check_24px.xml:6: error: resource color/white (aka org.wordpress.mobile.ReactNativeGutenbergBridge:color/white) not found.
  ...
```